src/Service/EnforcementManager.php

```diff
@@ -73,6 +74,28 @@ public function shouldRedirectToMFA(Member $member)
         return false;
     }
 
+    /**
+     * Check if the provided member has registered the required MFA methods. This includes a "back-up" method set in
+     * configuration plus at least one other method.
+     * Note that this method returns true if there is no backup method registered (and they have one other method
+     *
+     * @param Member&MemberExtension $member
+     * @return bool
+     */
+    public function hasFullyRegisteredMFA(Member $member)
```

Comment:
The method name irks me a little. How would you feel about `hasCompletedMFARegistration()` instead?